libs/openFrameworks/utils/ofSystemUtils.cpp

```diff
@@ -753,131 +730,6 @@ string ofSystemTextBoxDialog(string question, string text){
 		 DestroyWindow(dialog);
 		 EnableWindow(WindowFromDC(wglGetCurrentDC()), TRUE);
 
-	#else
-
```

Comment:
Not only, there's another subtle problem that one can notice: dependency on local width size, a variable that can't be synchronized with preferences of others. The simplest example is here, before our eyes: a guy who wrote this code set tabs to be 4 spaces, the Github interpreted them as 8. One may say "it is a problem of Github, it should change 8 spaces to 4", the other may say "it is a problem of this guy who wrote the code, he should change 4 spaces to 8", the third one will say "don't mix spaces with tabs", and so on, but no one can say anything about spaces -- their width is strictly defined and set in stone, everyone interpret them and see equally.